data/tribes/initialization/empire/units.lua

```diff
@@ -1445,6 +1540,10 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_arena",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire training site: Arena
+            lore = pgettext("empire_building", "‘Panem et circenses’ (Bread and games)"),
+            -- TRANSLATORS: Lore author helptext for an empire training site: Arena
+            lore_author = pgettext("is all it takes to keep the tribe happy"),
```

Comment:
Although one of the most famous Latin quotes, this might be misleading here, as it could be interpreted as a tip that bakeries and arenas are the most important buildings in the game.

Reply:
How about the revised author text?